### PR TITLE
Remove redundant `Helmet`s

### PR DESCRIPTION
**Pull request by ceolson01**
## Description

This PR removes some redundant usages of the `Helmet` component (cases where only `title` was being used, which exists in the `SEO` component already).

**Deletions** ⚰️

* Removed uses of the `Helmet` component where:
  * Only the `title` was being set
  * `SEO` was being used (with the same title as above ☝️)

### CODE REVIEW

File: src/modules/parts/components/SEO/SEO.jsx [hunk 1]
```diff
@@ -86,7 +106,7 @@ const SEO = ({
   }
 
   return (
-    <Helmet>
+    <Helmet title={title}>
```
Comment (ryanchristo) [inline src/modules/parts/components/SEO/SEO.jsx]: We might want to keep the comment about why `title` needs to be a prop.

```
Helmet title must be a prop to work with react hooks.
See https://github.com/nfl/react-helmet/issues/437
```

File: src/modules/parts/components/SEO/SEO.jsx [hunk 2]
```diff
@@ -17,18 +19,36 @@ const MSG = defineMessages({
...
+              ? files[`${project}/${imagePath}`]
+              : imagePath
+          }`;
+    },
+    [files, project],
+  );
+
+  const siteLogo = getAbsoluteImagePath('/img/colony-twitter-image.png');
+  const images = imagesProp || [siteLogo];
+
```
Comment (ryanchristo) [inline src/modules/parts/components/SEO/SEO.jsx:51]: Nice! All this looks much nicer here. :muscle:

**Review by ryanchristo — APPROVED** [review]
LGTM! One minor request regarding a comment. Preapproved. :100:

File: src/modules/parts/components/SEO/SEO.jsx (hunk 1 above)
Comment (ceolson01) [reply src/modules/parts/components/SEO/SEO.jsx]: Ah yes, good call 👍

**Pull request merged**